CodeiumVS/LanguageServer/LanguageServer.cs

```diff
@@ -200,7 +201,8 @@ public async Task SignOutAsync()
     /// <returns></returns>
     private async Task GetLanguageServerInfoAsync()
     {
-        string extensionBaseUrl = "https://github.com/Exafunction/codeium/releases/download";
+        string extensionBaseUrl = (_package.SettingsPage.ExtensionBaseUrl.Equals("") ? "https://github.com/Exafunction/codeium/releases/download"
+                                                                 : _package.SettingsPage.ExtensionBaseUrl);
```

Comment:
Just in case, to make it less prone to user input errors.
```suggestion
                                                                 : _package.SettingsPage.ExtensionBaseUrl.Trim().TrimEnd('/'));
```